File: ANGULARPROJECTS/ejemploangular03/src/app/Models/departamento.ts
export default class Departamento{
    public idDepartamento : number;
    public nombre : string;
    public localidad : string;

    constructor(idDepartamento ?:  number, nombre ?: string ,localidad ?: string ){
        this.idDepartamento =  idDepartamento ?? NaN;
        this.nombre = nombre ?? '';
        this.localidad = localidad ?? '';    
    }
}

File: .NET/AWS HOST/WebApplicationAWS/WebApplicationAWS/Models/Departamento.cs
﻿using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApplicationAWS.Models
{
    [Table("DEPT")]
    public class Departamento
    {
        [Key]
        [Required]
        [Column("DEPT_NO")]
        public int Id { get; set; }

        [Column("DNOMBRE")]
        public string? Nombre { get; set; }

        [Column("LOC")]
        public string? Localidad { get; set; }
    }
}